apps/acrs/behaviours/save-form-session.js

```diff
@@ -59,6 +59,10 @@ module.exports = superclass => class extends superclass {
           return res.redirect('/acrs/information-saved');
         }
 
+        if(req.body['exit']){
+          return res.redirect('/acrs/sign-in');
```

Comment:
The very first page of the app is on route `res.redirect('/sign-in')` (just needs the acrs bit taken out).